splitio/sync/synchronizer.py

```diff
@@ -225,7 +238,8 @@ def __init__(self, split_synchronizers, split_tasks):
         self._split_tasks = split_tasks
         self._periodic_data_recording_tasks = [
             self._split_tasks.impressions_task,
-            self._split_tasks.events_task
+            self._split_tasks.events_task,
+            self._split_tasks.telemetry_task
```

Comment:
do we need this reference here? otherwise we can just remove it and then we don't have to check for it when iterating during shutdown.

Reply:
We need it when we start services, i can remove it from here and add a line in start services call